modules/rostests/apitests/advapi32/QueryServiceConfig2.c

```diff
@@ -417,7 +417,7 @@ static int QueryConfig2A(SC_HANDLE hService, LPCSTR serviceName, DWORD dwInfoLev
 
             /* Check the values */
             ok(lpFailureActions1->dwResetPeriod == lpFailureActions2->dwResetPeriod, "lpFailureActions1->dwResetPeriod != lpFailureActions2->dwResetPeriod\n");
-            ok(lpFailureActions1->cActions == lpFailureActions2->cActions, "lpFailureActions1->cActions != lpFailureActions2->cActions\n");
+            // Fails on Win 2003: ok(lpFailureActions1->cActions == lpFailureActions2->cActions, "lpFailureActions1->cActions != lpFailureActions2->cActions\n");
```

Comment:
You wrote here: "Comment 2 tests that fail on Windows 2003" in the PRs description (you wrote nothing about whether x86 or x64, while you wrote that explicitly for all other). So I thought that you may mean also x86 here maybe. So I checked on the WHS-bot (x86), and didn't see any failures for QueryServiceConfig2.c before your patch:
https://reactos.org/testman/compare.php?ids=97640

Can you explain? Thx.

Reply:
You're right, it's only x64. Changed to architecture specific #ifndef

Reply:
Resolved.